buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -355,37 +361,40 @@ def get_summary_table() {
             // table cell
             def innerTable = "<table><tbody>"
 
-            def pipelineName = get_pipeline_name(spec, release)
-            def build = pipelineBuilds.get(pipelineName)
-            def downstreamJobNames = buildFile.get_downstream_job_names(spec, release)
+            // check if this release is supported for this spec
+            if (BUILD_SPECS.get(spec).contains(release)) {
```

Comment:
This change doesn't seem to fit the description of the PR. Should it be made, and described, in a separate PR?

Reply:
moved to another PR: see https://github.com/eclipse/openj9/pull/5376